libr/core/cmd.c

```diff
@@ -1821,6 +1821,16 @@ static int cmd_resize(void *data, const char *input) {
 			}
 		}
 		return true;
+	case 'j': // "rj"
+		if (core->file) {
+			PJ * pj = pj_new ();
+			pj_o (pj);
+			if (oldsize != -1) {
+				pj_n(pj, oldsize);
+			}
+			pj_end (pj);
+			pj_free (pj);
```

Comment:
Drain+print+free the pj string

Reply:
How about
```
		char *s = pj_drain (pj);
		if (s) {
			r_cons_println (s);
			free (s);
		}
```
after the pj_end?

Reply:
if you call pj_drain you dont need to call pj_free,